lib/Twig/Environment.php

```diff
@@ -117,6 +119,40 @@ public function __construct(Twig_LoaderInterface $loader = null, $options = arra
     }
 
     /**
+     * Returns deprecations for templates contained in a directory.
+     *
+     * @param string $dir The directory where templates are stored
+     * @param string $ext Limit the loaded templates by extension
+     *
+     * @return An array of deprecations
```

Comment:
``@return array`` is needed, otherwise the phpdoc is invalid and IDEs cannot use it (or even better ``@return string[]``)